QuizApp/QuizApp/Main/AppRouter.swift

```diff
@@ -34,6 +34,16 @@ class AppRouter: AppRouterProtocol {
         navigationController.setViewControllers([tabBarController], animated: true)
     }
 
+    func showQuizDetails(with quiz: Quiz) {
+        let quizDetailsViewController = container.resolve(QuizDetailsViewController.self, args: quiz)
+
+        navigationController.pushViewController(quizDetailsViewController, animated: true)
+    }
+
+    func popViewController() {
```

Comment:
`func goBack()`
Does someone need to know how exactly does router go back? No, the caller just wants to go back, it does not care about the implementation ;)